firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/ApkUpdaterTest.java

```diff
@@ -169,6 +169,35 @@ public void updateApk_whenInstallSuccessful_setsResult() throws Exception {
     assertThat(updateTask.isSuccessful()).isTrue();
   }
 
+  @Test
+  public void updateApk_getForeGroundActivityFails_setsError() {
+    boolean showNotification = true;
+    doReturn(Tasks.forResult(mockFile))
+        .when(apkUpdater)
+        .downloadApk(TEST_RELEASE, showNotification);
+    TaskCompletionSource<Activity> installTaskCompletionSource = new TaskCompletionSource<>();
```

Comment:
Can you change the name of this to something like `getForegroundActivityCompletionSource`?